### DISCUSSION TITLE
plusieurs plots sur plusieur ordinateurs +> pour meme ferme ?

### DISCUSSION THREAD

**Discussion by Liopal** [Chia-Network/chia-blockchain#7755] [Support]
Bonjour j'ai actuellement 2 pc qui tourne avec la même clé 
sur les 2 pc il y a des plots , 
le soucis c'est que je n'arrive pas à les ratacher à la meme ferme je vois que les stat du pc actuel 
ne serrais ce pas possible de les cumuler sur la meme ferme ? / ou comment le parramêtrer ? merci

**Comment by keliew** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
